CallAutomation_CallLiveTranscription/CallAutomationLiveTranscription/Helper.cs

```diff
@@ -14,8 +14,8 @@ public static async Task ProcessRequest(WebSocket webSocket)
         {
             try
             {
-                while (webSocket.State == WebSocketState.Open || webSocket.State == WebSocketState.CloseSent)
-                {
+               // while (webSocket.State == WebSocketState.Open || webSocket.State == WebSocketState.CloseSent)
```

Comment:
Why we have commented this? this might still need it

Reply:
And please test and have the correlation id along with PR description

Reply:
this while loop does not require, its needs to be removed, otherwise its will goes continually into loop, previous added wrongly, so i removed and tested, even earlier WebSocket code also don't gave this while loop.

Reply:
CALL CONNECTION ID--?0b004480-149a-49c4-8fda-8341cb41d541
CORRELATION ID-->3f47e74f-5b32-4e18-aa1d-c49e40f03c4d